create_test_data.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+from sqlalchemy import orm
+import string
+
+from fidesops.core.config import config
+from fidesops.db.database import init_db
+from fidesops.db.session import get_db_session
+from fidesops.models.client import ClientDetail
+from fidesops.models.fidesops_user import FidesopsUser
+from fidesops.models.policy import ActionType, Policy, Rule, RuleTarget
+from fidesops.models.privacy_request import PrivacyRequest, PrivacyRequestStatus
+from fidesops.util.data_category import DataCategory
+
+
+"""Script to create test data for the Admin UI"""
+
+
+def _create_policy(
+    db: orm.Session,
+    action_type: str,
+    client_id: str,
+) -> Policy:
+    rand = string.ascii_lowercase[:5]
+    policy = Policy.create(
+        db=db,
+        data={
+            "name": f"example {action_type} policy {rand}",
```

Comment:
Could we also add a `download` and `delete` policy to this function? The privacy center uses these policies by default and they need to be recreated each time the database is reset. I found myself doing that a few times while developing frontend.

Reply:
By `download` do you mean `access`?

Reply:
No, I do mean `download` and `delete`. It looks like the privacy center has a config file set up to use those as the policy keys. Here is the mentioned [file](https://github.com/ethyca/fidesops/blob/main/clients/privacy-center/config/config.json#L7-L30).

```json
  "actions": [
    {
      "policy_key": "download",
      "icon_path": "/download.svg",
      "title": "Download your data",
      "description": "We will email you a report of the data related to your account.",
      "identity_inputs": {
        "name": "optional",
        "email": "required",
        "phone": "optional"
      }
    },
    {
      "policy_key": "delete",
      "icon_path": "/delete.svg",
      "title": "Delete your data",
      "description": "We will delete all of your account data. This action cannot be undone.",
      "identity_inputs": {
        "name": "optional",
        "email": "required",
        "phone": "optional"
      }
    }
  ]
```

Reply:
Are `access` and `erasure` policy keys seeded already?

Reply:
> Are access and erasure policy keys seeded already?

They aren't